readme.md

```diff
@@ -93,6 +93,13 @@ Default: `true`
 
 A reasonable amount of effort will be spent to try to recover as much data as possible of a broken image, but the success cannot generally be guaranteed.
 
+##### strip
```

Comment:
```suggestion
##### stripMetadata
```

Reply:
but wouldn't that strip metadata? Or would that be too simple

Reply:
I'm not sure what you're asking. I'm saying the option should be named better as it only strips metadata.